mkat_fpga_tests/__init__.py

```diff
@@ -25,6 +30,18 @@ def __init__(self, test_config_filename=None):
         self._correlator = None
         self._dhost = None
 
+        self.io_manager = ioloop_manager.IOLoopManager()
```

Comment:
I'll merge this for now, but generally side-effects (such as connecting to network devices) in an __init__ it not a good idea.

We also need to figure out how to do proper cleanup.